consensus/propagation/reactor.go

```diff
@@ -99,7 +100,8 @@ func NewReactor(
 		chainID:       config.ChainID,
 		BlockMaxBytes: config.BlockMaxBytes,
 		partChan:      make(chan types.PartInfo, 30_000),
-		proposalChan:  make(chan types.Proposal, 100),
+		proposalChan:  make(chan ProposalAndFrom, 1000),
+		ticker:        time.NewTicker(RetryTime),
```

Comment:
adding the ticker here so that we reset it on every height so catchup is not triggered during the happy path